src/concepts/essentials/workflows/js_in_caido.md

```diff
@@ -0,0 +1,499 @@
+# JavaScript in Caido
+
+_Below includes in-depth foundational information, to skip to usage of JavaScript in Workflow Nodes - navigate to the [JavaScript Node Functions](#javascript-node-functions) section._
+
+## Why JavaScript?
+
+Caido's decision to implement JavaScript as opposed to another programming language was arrived at based on multiple factors.
+
+With JavaScript, context switching between the frontend and backend is minimal. JavaScript is a versitile language and the speed performance is noticeable in the Caido environment. Also, as an HTTP proxy, JavaScript is a familiar language to those that are using Caido as it is present in every engagement.
```

Comment:
- `Versatile`
- `the speed performance is noticeable in the Caido environment` -> `easy to learn`?
- Remove `Also, as an HTTP proxy,`?